tests/test_examples.py

```diff
@@ -235,11 +238,12 @@ def _run_example(example):
     class Timeout(Exception):
         pass
 
-    def alarm_handler(sig, frame):
-        raise Timeout
+    if not is_windows:
+        def alarm_handler(sig, frame):
+            raise Timeout
 
-    signal.signal(signal.SIGALRM, alarm_handler)
-    signal.alarm(20 if not example.is_slow else 60)
+        signal.signal(signal.SIGALRM, alarm_handler)
+        signal.alarm(20 if not example.is_slow else 60)
```

Comment:
Currently no timeouts on Windows. There are alternatives, but currently it's more work than it's worth.